sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/TrouterUtils.java

```diff
@@ -270,7 +282,8 @@ private static BaseEvent getReadReceiptReceived(JSONObject payload) {
             eventPayload.setThreadId(payload.getString("groupId"));
 
             eventPayload.setSender(getUserIdentifier(payload.getString("senderId")));
-            eventPayload.setRecipient(getUserIdentifier(payload.getString("recipientId")));
+            eventPayload.setRecipient(getUserIdentifier(
+                parseRecipientSkypeId(payload.getString("recipientId"))));
```

Comment:
we should not use the term `SkypeId` here. We tend to call it `RawId` in other places